core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.html

```diff
@@ -10,6 +10,15 @@
       </i>
     </span>
   </div>
+  <div class="story-node-field">
+    <label for="storyNodeDescription" class="form-heading">Chapter Description</label>
+    <input id="storyNodeDescription" type="text" class="form-control protractor-test-add-chapter-description"
```

Comment:
Also, change this to a <textarea> input with more height since this can be a detailed description (similar to topic description).

Reply:
Done.